config/application.rb

```diff
@@ -117,6 +117,9 @@ def config.database_configuration
 
     Rails.autoloaders.main.ignore(Dir["#{config.root}/app/models/reports"])
 
+    require_dependency 'lib/zeitwerk/zeitwerk.rb'
+    Rails.autoloaders.main.ignore(Dir["#{config.root}/lib/freedom_patches"])
```

Comment:
Instead of having to ignore this, can we move this from the autoload path by default?

Reply:
I moved the freedom_patches directory outside of lib and then we don't need that ignore 👍